src/util/index.ts

```diff
@@ -1,7 +1,7 @@
-import axios from "axios";
+import axios, { AxiosRequestConfig } from "axios";
  
 // note: not used, but could be used with GET with params
- const getData = async (url, params) => {
+ const getData = async (url: string, params: AxiosRequestConfig<any> | undefined) => {
```

Comment:
It's suggested to use `any` as little as possible. We won't be using this method in the future any way so it's fine at this point